src/modules/Common/components/SubscribeForm.tsx

```diff
@@ -79,12 +79,7 @@ const SubscribeForm = ({
 
   return (
     <form onSubmit={handleSubmit(onSubmit)}>
-      <h4 className="mb__L">
-        {t(
-          'common:subscribe_form.title',
-          'Be informed by email of the changes on the documents of your choice.'
-        )}
-      </h4>
+      <h4 className="mb__L">{t('subscribe_form.title', { ns: 'common' })}</h4>
```

Comment:
I do not understand why you need to specify the namespace here
In the example here https://github.com/isaachinman/next-i18next#usetranslation the namespace is passed directly to `useTranslation` which seems a lot neater to me.

I understand if we use several namespaces in the same component (which should maybe makes us ask a question on whether components should not be splitted)

Reply:
Also, I believe it's the same effect than `common:subscribe_form.title` which is less verbose and more easy to write

Reply:
you are absolutely right, it is well noted, thanks 🙏